packages/radix-ui-themes/src/props/layout.props.ts

```diff
@@ -143,31 +322,10 @@ const layoutPropDefs = {
   gridRowEnd: PropDef<string>;
 };
 
-const deprecatedLayoutPropDefs = {
-  /** @deprecated Rename this prop to `flexShrink`. The `shrink` prop will be removed in the next major release. */
-  shrink: {
-    type: 'enum | string',
-    className: 'rt-r-fs',
-    customProperties: ['--flex-shrink'],
-    values: flexShrinkValues,
-    default: undefined,
-    responsive: true,
-  },
-  /** @deprecated Rename this prop to `flexGrow`. The `grow` prop will be removed in the next major release. */
-  grow: {
-    type: 'enum | string',
-    className: 'rt-r-fg',
-    customProperties: ['--flex-grow'],
-    values: flexGrowValues,
-    default: undefined,
-    responsive: true,
-  },
-} satisfies {
-  shrink: PropDef<(typeof flexShrinkValues)[number]>;
-  grow: PropDef<(typeof flexGrowValues)[number]>;
-};
-
-type LayoutProps = GetPropDefTypes<typeof layoutPropDefs & typeof deprecatedLayoutPropDefs>;
+// Use all of the imported prop defs to ensure that JSDoc works
+type LayoutProps = GetPropDefTypes<
+  typeof paddingPropDefs & typeof widthPropDefs & typeof heightPropDefs & typeof layoutPropDefs
+>;
```

Comment:
JSDoc won't work when we are just spreading padding, width, height, etc. prop defs into the layout pro defs; however it does work if we spread AND include the types here. This is the same reason `asChildProp` was changed to be an object—so that we can do `& typeof asChildPropDef`